docs/index.html

```diff
@@ -0,0 +1,519 @@
+      }
+      @keyframes highlight_glow {
+        0% {
+          background: none;
+        }
+        10% {
+          background: var(--light);
+        }
+        100% {
+          background: none;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <section>
+        <div class="section">
+          <div class="logo">
+            <img src="logo.svg" />
+          </div>
+          <nav>
+            <a href="#summary">Summary</a>
+            <a href="#citation">Citation</a>
+            <a href="#approach">Approach</a>
+            <a href="#resources">Resources</a>
+            <a href="#acknowledgements">Acknowledgements</a>
+          </nav>
+        </div>
+      </section>
+    </header>
+    <main>
+      <section id="summary">
+        <div class="section">
+          <h2>Summary</h2>
+          <p>
+            BioBombe represents an approach to optimize extraction of biological
+            representations in gene expression data. Named after the large
+            mechanical devices built by Alan Turing and other cryptologists
+            during World War II to decode secret messages, BioBombe is used to
+            decipher hidden messages embedded in gene expression data. By
+            applying multiple common dimensionality reduction algorithms across
+            a wide range of latent dimensionalities, we use the BioBombe
+            approach to study which algorithms and latent dimensionalities best
+            capture certain biological signal.
+          </p>
+          <p>
+            This website provides convenient links to all the resources provided
+            by the manuscript including software, processed gene expression
+            input data, compressed features for all algorithms, latent
+            dimensionalities, and initializations for all datasets, and
+            performance results and models for the large scale cancer-type and
+            mutation classification analysis.
+          </p>
+          <h3>Primary Findings</h3>
+          <p>
+            <i
+              >There does not exist a single optimal algorithm or latent
+              dimensionality for learning biological representations</i
+            >. Many different biological signatures are learned by different
+            compression algorithms at various latent dimensionalities. A
+            practitioner aiming to optimize feature discovery by compressing
+            gene expression data should use multiple algorithms across a large
+            range of latent dimensionalities.
+          </p>
+        </div>
+      </section>
+      <section id="citation">
+        <div class="section">
+          <h2>Citation</h2>
+          <p>
+            <b
+              >Sequential compression across latent space dimensions enhances
+              gene expression signatures</b
+            ><br />
+            Way, G.P., Zietz, M., Himmelstein, D.S., Greene, C.S.<br />
```

Comment:
Create an issue to add @vincerubinetti when the new preprint is posted.

Reply:
#182